codespell_lib/data/dictionary.txt

```diff
@@ -6455,6 +6456,7 @@ attributring->attributing
 attributrion->attribution
 attributrions->attributions
 attributs->attributes
+attributse->attributes
```

Comment:
I wonder whether this could be a typo of singular `attribute`, given that key `s` is underneath key `e` on many keyboards:
```suggestion
attributse->attributes, attribute,
```